skipruntime-ts/examples/database-client.ts

```diff
@@ -27,13 +31,7 @@ const [_e, headers] = await fetchJSON<ReactiveResponse>(
   header,
 );
 
-const strReactiveResponse = headers.get("x-reactive-response");
-const reactive = strReactiveResponse
-  ? (JSON.parse(strReactiveResponse, (key: string, value: string) => {
-      if (key == "watermark") return BigInt(value);
-      return value;
-    }) as ReactiveResponse)
-  : undefined;
+const reactive = parseReactiveResponse(headers);
```

Comment:
Tangential to this PR, but this makes me think that deconstruction of the `Reactive-Response-Token` should happen within the skip client, and the token itself be passed transparently through to `subscribe()`, instead of `subscribe()` taking a collection name + watermark. Just something to keep in mind, no action required on your end in this PR.